### ISSUE TITLE
Add option to disable all filtering, custom filtering or custom block/allow lists

### ISSUE THREAD

**Issue by itsnotsaved** [AdguardTeam/AdGuardHome#4508]
### Problem Description
If disable block domains using filters and hosts files then it disable everything (no filtering); no way to keep default blocklists enabled

### Proposed Solution

Add option to general settings for disable,
- [x] All filtering (available)
  - [ ] Custom filtering rules
  - [ ] Custom block lists
  - [ ] Custom allow lists
  - [ ] Blocked services

**Comment by fernvenue** [comment]
I don't think so, the reason this option is in **general settings** is because it applies to all lists, and that's why they put it in general settings. We can also enable or disable any one or more lists in DNS blocklists and DNS allowlists, in custom rules, we can also use the form of comment to keep the rule but make it not enabled. So I think we don't need more options to do that.

**Comment by itsnotsaved** [comment]
@fernvenue I tried to handle this for years but because of keeping public strict lists, custom made lists for different purposes (such as devices updates (OSes, phones), software updates), various custom filtering rules; it's painful to enable / disable sometimes or search and comment out particular rule in custom filtering rules. 
My expectation is to keep enabled adguardhome itself ship default blocklists so need a easy way to disable custom things

**Comment by fernvenue** [comment]
@itsnotsaved Sounds reasonable, maybe adding three sub-options would be more convenient when using so many rules.
